lhotse/augmentation/torchaudio.py

```diff
@@ -212,6 +212,174 @@ def reverse_timestamps(
         )
 
 
+class Codec:
+    def __call__(self, samples: np.ndarray) -> np.ndarray:
+        """
+        Apply encoder then decoder.
+
+        To be implemented in derived classes.
+        """
+        raise NotImplementedError
+
+
+class MuLawCodec:
+    def __init__(self):
+        import torchaudio
+
+        self.encoder = torchaudio.transforms.MuLawEncoding()
+        self.decoder = torchaudio.transforms.MuLawDecoding()
+
+    def __call__(self, samples):
+        return self.decoder(self.encoder(samples))
+
+
+from ctypes import CDLL, POINTER, c_int, c_short, c_uint8, c_void_p
+
+LPC10_FRAME_SAMPLES = 180
+LPC10_FRAME_BYTES = 7
+
+
+def libspandsp_api():
```

Comment:
Can we gracefully handle errors here by raising an exception with a clear message such as "We can't apply LPC10 codec because we couldn't locate library libspandsp.so. In order to use this augmentation, do ... "